scripts/clean_osm_data.py

```diff
@@ -399,6 +403,10 @@ def integrate_lines_df(df_all_lines, distance_crs):
     else:
         df_all_lines["tag_frequency"] = ac_freq_default
 
+    # it may happen that the frequency column also contains ";"
+    df_all_lines["tag_frequency"] = df_all_lines["tag_frequency"].astype(str)
+    df_all_lines = split_cells(df_all_lines, lst_col="tag_frequency")
```

Comment:
I think that the split_cells used here is improper as we duplicate line entries that may have the ";" in also the cables and circuits.
The tag_frequency should be added into the split_cells_multiple, but that function should be heavily improved to account for semicolumn partially missining in some columns.

In the case where semicolumn is found in tag_frequency, are the values consistent (e.i. "50;50") or not (e.i. ("50;0")?